parmed/openmm/parameters.py

```diff
@@ -372,6 +390,6 @@ def _write_omm_scripts(self, dest):
                                       'supported.')
         if len(self.nbfix_types) > 0:
             raise NotImplementedError('NBFIX not currently supported')
-        if self.urey_bradley_types:
-            raise NotImplementedError('Urey-Bradley angles not currently '
-                                      'supported.')
+        # if self.urey_bradley_types:
+        #     raise NotImplementedError('Urey-Bradley angles not currently '
+        #                               'supported.')
```

Comment:
Let's just delete these lines.  No reason to keep them commented out, since there's no reason to ever un-comment them back in.